traffic_ops/traffic_ops_golang/invalidationjobs/invalidationjobs.go

```diff
@@ -168,11 +238,35 @@ INNER JOIN deliveryservice ON deliveryservice.id=job.job_deliveryservice
 WHERE job.id=$1
 `
 
+// Deprecated, only to be used with versions below 4.0
 const deleteQuery = `
 DELETE
 FROM job
 WHERE job.id=$1
 RETURNING job.asset_url,
+          ( SELECT tm_user.username
+           FROM tm_user
+           WHERE tm_user.id=job.job_user
+          ) AS created_by,
+          (
+           SELECT deliveryservice.xml_id
+           FROM deliveryservice
+           WHERE deliveryservice.id=job.job_deliveryservice
+          ) AS deliveryservice,
+          job.id,          
+		  'PURGE' as keyword,
+          CONCAT('TTL:', ttl_hr, 'h') AS parameters,
+          job.start_time
+`
+
+// Almost the same as deleteQuery, but returns appropriate values for API 4.0+
+const deleteQueryV40 = `
```

Comment:
We should probably call this V4 instead of V40, because we'll likely just update this query with new fields if they're ever added.